aws_tools/dynamodb_handler.py

```diff
@@ -77,48 +77,53 @@ def delete_item(self, keys):
         )
 
     def query_items(self, query=None, only_fields_with_values=True):
-        filter_expression = ''
+        filter_expression = None
         if query and len(query) > 1:
             for field, value in iteritems(query):
-                value = None
                 value2 = None
                 if isinstance(value, dict) and 'condition' in value and 'value' in value:
-                    condition = value['condition']
+                    condition_str = value['condition']
                     value = value['value']
-                    if 'value2' in value:
-                        value2 = value['value2']
+                    if condition_str == "between":
+                        value, value2 = value["value"], value["value2"]
                 else:
-                    condition = 'eq'
+                    value = None
+                    condition_str = 'eq'
 
                 if not value and only_fields_with_values:
                     continue
 
-                if condition == 'eq':
-                    filter_expression &= Attr(field).eq(value)
-                if condition == 'ne':
-                    filter_expression &= Attr(field).ne(value)
-                if condition == 'lt':
-                    filter_expression &= Attr(field).lt(value)
-                if condition == 'lte':
-                    filter_expression &= Attr(field).lte(value)
-                if condition == 'gt':
-                    filter_expression &= Attr(field).gt(value)
-                if condition == 'gte':
-                    filter_expression &= Attr(field).gte(value)
-                if condition == 'begins_with':
-                    filter_expression &= Attr(field).begins_with(value)
-                if condition == 'between':
-                    filter_expression &= Attr(field).between(value, value2)
-                if condition == 'ne':
-                    filter_expression &= Attr(field).ne(value)
-                if condition == 'is_in':
-                    filter_expression &= Attr(field).is_in(value)
-                if condition == 'contains':
-                    filter_expression &= Attr(field).contains(value)
+                if condition_str == 'eq':
+                    condition = Attr(field).eq(value)
```

Comment:
We can't use `filter_expression &= ...` here, because `filter_expression` could be `None`.